### PR TITLE
Padding before deleting PVC's

### PR DESCRIPTION
**Pull request by cxhercules**

### CODE REVIEW

**Comment by cxhercules** [comment]
@chrislovecnm I added some padding besides the grace period. I am not certain this helped any as I have not seen the failure scenario play out. Also if concurrent builds where allowed this could have added to issue, but I see you are disabling that now. Another thing that came to mind is using `gcloud compute disks list --project pso-helmsman-cicd-infra --filter="name~'gke-staging-on-prem-cl-pvc*'"` to gather list and pass that on to `gcloud compute disk delete` as a secondary process if needed.

**Comment by chrislovecnm** [comment]
I think deleting the storage class before the pvc’s might be causing problems as well. A guess.

**Comment by chrislovecnm** [comment]
I do not want to use a work around, we need to figure out why this is happening.

**Comment by chrislovecnm** [comment]
@cxhercules I am going to run the test once more, but I think this was it.  Jenkins is just running too fast.  Can one of use check the test infra project and see if we have hanging disks?

File: scripts/on-prem-destroy.sh [hunk 1]
```diff
@@ -39,14 +39,21 @@ if [[ ! $(kubectl config use-context "$context") ]]; then
 fi
 
 # You have to wait the default pod grace period before you can delete the pvcs
-grace=$(kubectl --namespace default get sts -l component=elasticsearch,role=data -o jsonpath='{..terminationGracePeriodSeconds}')
+GRACE=$(kubectl --namespace default get sts -l component=elasticsearch,role=data -o jsonpath='{..terminationGracePeriodSeconds}')
+PADDING=120
```
Comment (chrislovecnm) [inline scripts/on-prem-destroy.sh]: is this 120 seconds?  I am wonder if another 30 would be good enough.

Comment (cxhercules) [reply scripts/on-prem-destroy.sh]: It is definitely worth trying to find the sweet spot. I will update.

**Comment by chrislovecnm** [comment]
Need a rebase as well please

**Comment by cxhercules** [comment]
Updated and rebased.

**Comment by chrislovecnm** [comment]
CI failed

**Comment by cxhercules** [comment]
CI failed because of container issue that I spoke to @homernajafi and @pydevops about. I will request testing again.

**Comment by cxhercules** [comment]
Test this please

**Comment by cxhercules** [comment]
@chrislovecnm CI has been verified 2 since failure that was due to container issue, and verified no disks left behind.

**Pull request merged**